### PR TITLE
Uditkumar01/checklist bug fixes

### PR DESCRIPTION
**Pull request by uditkumar01**
### Fixes
* **Scrolling** issue
* **redirecting user to dashboard** when submission status is under review

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -48,15 +48,15 @@ app.use((err: Error, req: Request, res: Response, next: NextFunction) => {
...
-// }
+if (process.env.NODE_ENV === 'development') {
+  app.listen(process.env.PORT, () => {
+    log.info(`[server] ->🚀 started on ${process.env.PORT}`)
+    log.warn(`⚠️ Make sure to use "yarn serve" in production.`)
+    log.info(
+      `[${process.env.NODE_ENV}] -> http://localhost:${process.env.PORT}`
+    )
+  })
+}
```
Comment (SAYUK09) [inline src/app.ts:59]: I guess we aren't supposed to push this file to GitHub right?
Won't this cause merge conflicts and break production?

File: client/next.config.js [hunk 2]
```diff
@@ -5,6 +5,9 @@ module.exports = {
     ignoreDuringBuilds: true,
   },
   env: {
-    API_URL: 'https://neog-admissions.azurewebsites.net',
+    API_URL: 'http://localhost:5000', // for local development
```
Comment (SAYUK09) [inline client/next.config.js:8]: Do we need to change the `API_URL` back to `https://neog-admissions.azurewebsites.net` before pushing the code?

**Review by SAYUK09 — CHANGES_REQUESTED** [review]
Everything seems good. Just have few doubts, commented them.

**Comment by uditkumar01** [comment]
Yes, we need to make it compatible with both dev and prod somehow. As it's really cumbersome to change it every time before making a pr.

File: client/next.config.js (hunk 2 above)
Comment (sushilburagute) [reply client/next.config.js:8]: Oh yes. the `API_URL` needs to be that

**Comment by uditkumar01** [comment]
Fixed in #191

**Pull request closed without merging**